nixnet/_session/base.py

```diff
@@ -335,6 +335,37 @@ def disconnect_terminals(self, source, destination):
         """
         _funcs.nx_disconnect_terminals(self._handle, source, destination)
 
+    def write_state_lin_schedule_change(self, state_value):
+        # type: () -> None
+        """Writes communication states of an XNET session.
+
+        This function writes a request for the LIN interface to change
+        the running schedule.
+
+        According to the LIN protocol, only the master executes schedules,
+        not slaves. If the
+        :any:`nixnet._session.intf.Interface.lin_master` property is false (slave),
+        this write function implicitly sets that property to true (master). If the
+        interface currently is running as a slave, this write returns an error,
+        because it cannot change to master while running.
+
+        Args:
+            state_value(str): Writes the desired state.
+        """
+        _funcs.nx_write_state(self._handle, constants.WriteState.LIN_SCHEDULE_CHANGE, state_value)
+
+    def write_state_lin_diagnostic_schedule_change(self, state_value):
+        # type: () -> None
```

Comment:
Also, since this will be using a yet-to-be-documented enum, this change should also include documenting the enum.

You can take the documentation from nxWriteState for the values and document the `LinDiagnosticSchedule` class similarly to how we documented the other enums.